examples/locks/locks.bal

```diff
@@ -0,0 +1,37 @@
+import ballerina/io;
+
+// Shared counter variable among multiple workers.
+int counter;
+
+function main (string... args) {
```

Comment:
This is better to include the loops inside the workers itself, so it clearly shows that concurrently the variable will be incremented in each worker. In the current way, there is like a change for each "counterUp" call, that there will be a sequential delay in the order the workers are initialized, so the variable access may not happen concurrently.